tests/scripts/t3000/run_t3000_frequent_tests.sh

```diff
@@ -60,13 +60,19 @@ run_t3000_llama3_tests() {
   # Llama3.2-11B
   llama11b=/mnt/MLPerf/tt_dnn-models/llama/Llama3.2-11B-Vision-Instruct/
 
+  # Install Vision-specific packages
+  pip install -r models/demos/llama3/requirements.txt
```

Comment:
Why does it have to be separate? I don't like this - we should contain all our dependencies in one go. Remember we cache python environments

Reply:
Following a discussion with @tt-rkim, created a P1 issue (https://github.com/tenstorrent/tt-metal/issues/16235) to address this separately since it is an existing issue for llama3 jobs in other pipelines as well.